.circleci/windows/build.cmd

```diff
@@ -22,10 +22,12 @@ IF NOT "%ARTIFACTS_PATH%"=="" FOR /f %%i IN ('cygpath -a %ARTIFACTS_PATH%') DO S
 
 IF "%PACKAGES_PATH%"=="" SET "PACKAGES_PATH=%PROJECT_ROOT%/packages"
 
-call "%~dp0vcvarsall" || exit /b %ERRORLEVEL%
+call "%~dp0vcvarsall"
+IF ERRORLEVEL 1 exit /b %ERRORLEVEL%
```

Comment:
```suggestion
IF %ERRORLEVEL% NEQ 0 exit /b %ERRORLEVEL%
```

Reply:
I think this is more appropriate

Reply:
Seems I missed the fact that ERRORLEVEL is resolved before the actual execution
so it had to be
```
SETLOCAL EnableDelayedExpansion

command-1 || exit /b !ERRORLEVEL!
```
However it requires to have setlocal which is not put everywhere on purpose (because it creates a new SET scope and thus can not work with the global scope, which is required on few places)

Reply:
Correct. But I also like the check on a separate line as it is more explicit. I am fine with using your suggestions.